Gems/Atom/RPI/Code/Include/Atom/RPI.Public/View.h

```diff
@@ -158,10 +158,10 @@ namespace AZ
             //! Value returned is 1.0f when an area equal to the viewport height squared is covered. Useful for accurate LOD decisions.
             float CalculateSphereAreaInClipSpace(const AZ::Vector3& sphereWorldPosition, float sphereRadius) const;
 
-            const AZ::Name& GetName() const { return m_name; }
-            const UsageFlags GetUsageFlags() { return m_usageFlags; }
+            const AZ::Name& GetName() const;
+            const UsageFlags GetUsageFlags();
```

Comment:
The UsageFlags return type should be marked `const`.

Also the member function itself should be const

This should get flagged on a clang compiler.
```suggestion
            UsageFlags GetUsageFlags() const;
```